.github/workflows/nipoppy_track_processing_for_dataset.yaml

```diff
@@ -0,0 +1,120 @@
+name: Generate processing status files for OpenNeuroDatasets
+
+on:
+  workflow_dispatch:
+    inputs:
+      chunk_size:
+        description: "Number of datasets to process in each job"
+        required: false
+        default: "100"
+
+jobs:
+  define-matrix:
+    name: Get chunks of dataset IDs to run on
+    runs-on: ubuntu-latest
+    outputs:
+      dataset_id_chunks: ${{ steps.chunk-datasets.outputs.dataset_id_chunks }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+
+      - name: Chunk dataset IDs from mapping file
+        id: chunk-datasets
+        run: |
+          chunk_size=${{ github.event.inputs.chunk_size }}
+
+          dataset_id_chunks=$(jq -c 'keys as $dataset_ids | \
+            [range(0; $dataset_ids|length; '"$chunk_size"') | \
+            $dataset_ids[. : . + '"$chunk_size"']]' \
+            derivatives/repo_derivatives_map.json)
+
+          echo "dataset_id_chunks=$dataset_id_chunks" >> "$GITHUB_OUTPUT"
+
+  generate-processing-status-files:
+    name: Generate processing status files
+    needs: define-matrix
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        dataset_id_chunk: ${{ fromJson(needs.define-matrix.outputs.dataset_id_chunks) }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: "3.11"
+
+      # TODO: Switch to using requirements.txt
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install nipoppy
+
+      - name: Create Nipoppy dataset
+        run: |
+          count=1
+          total_datasets=$(jq -r 'length' <<< "${{ matrix.dataset_id_chunk }}")
+          for dataset_id in $(jq -r '.[]' <<< "${{ matrix.dataset_id_chunk }}"); do
+            echo "($count/$total_datasets) Processing dataset: $dataset_id"
+            .github/code/create_nipoppy_dataset.sh "$dataset_id"
```

Comment:
I don't see `.github/code/create_nipoppy_dataset.sh` in this PR or in `main`. Did you mean `.github/code/nipoppy_track_processing_for_dataset.sh`?

Reply:
oops - yes, I renamed the script 😅 good catch!